src/app/features/collectibles/components/bitcoin/stamp.tsx

```diff
@@ -11,7 +13,7 @@ export function Stamp(props: { bitcoinStamp: BitcoinStamp }) {
 
   return (
     <CollectibleImage
-      icon={<StampsIcon size="30px" />}
+      icon={<StampsIcon size={token('icons.icon.lg')} />}
```

Comment:
I guess we still need `token` for this size, even with `Circle` from `leather-styles/jsx` wrapping the SVG